docker/server-init.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+./linera-db check_existence --storage "scylladb:tcp:scylladb.default.svc.cluster.local:9042"
+status=$?
+
+if [ $status -eq 0 ]; then
+  exit 0
+elif [ $status -eq 1 ]; then
+  ./linera-server initialize \
+    --storage scylladb:tcp:scylladb.default.svc.cluster.local:9042 \
```

Comment:
Maybe we could put `scylladb:tcp:scylladb.default.svc.cluster.local:9042` in a variable? Then just reference it in both places. To avoid typo issues :)

Reply:
Yes needs to be done at some point - we will probably end up updating the db name as well.